.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "linebreak-style": "off",
+      "import/prefer-default-export": "off",
+      "class-methods-use-this": "off",
+      "import/no-unresolved": "off",
+      "no-plusplus": "off"
+    }
+}
```

Comment:
Please do not make changes to this file ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)

Reply:
Hey,
I've tried every possible solutions to fix the linter issues but there are some options such as no-plusplus that needs to be disabled if the code is to work. It broke my app and I'm unfortunately unwilling to enable such options

Reply:
You can also see from the linter documentation that I'm allowed to make changes if necessary.
![linter](https://user-images.githubusercontent.com/20954163/77684519-2c354200-6fab-11ea-9ee6-bf216292bf86.PNG)

Reply:
I've had to make these changes because it will break my app